pfetch

```diff
@@ -612,6 +631,21 @@ get_memory() {
             mem_used=$(((mem_full - mem_free) / 1024))
             mem_full=$(( mem_full / 1024))
         ;;
+
+        SunOS)
+            hw_pagesize=$(pagesize)
+
+            while read -r _ p; do
+                : "${pages_total:+"${pages_free="$p"}"}" "${pages_total="$p"}"
+            done <<-EOF
+				$(kstat -p unix:0:system_pages:pagestotal \
```

Comment:
Full output of: 

```sh
kstat -p unix:0:system_pages:pagestotal unix:0:system_pages:pagesfree
```

Reply:
```
unix:0:system_pages:pagestotal	1046397
unix:0:system_pages:pagesfree	885018
```

Reply:
I think this is clearer: 

```sh
while read -r key val; do
    case $key in
        *total) mem_full=$val ;;
        *free)  mem_free=$val ;;
    esac
done <<-EOF
	$(kstat -p unix:0:system_pages:pagestotal \
	           unix:0:system_pages:pagesfree)
EOF
```

Reply:
(Will fix indentation when it hits master)

Reply:
You mean `pages_total` and `pages_free`?